app/api/events.py

```diff
@@ -168,7 +168,7 @@ def create(self, data, url):
         save_to_db(new_event, "Event saved")
         # set organizer
         role = Role.query.filter_by(name=ORGANIZER).first()
-        uer = UsersEventsRoles(g.user, new_event, role)
+        uer = UsersEventsRoles(user_id=creator_id, event=new_event, role=role)
```

Comment:
What you can do is check if creator_id is set or not. If not (i.e. None), use `g.user.id` in that place. This won't break the EventList.post method.